src/language-js/print/ternary.js

```diff
@@ -153,6 +153,105 @@ function printTernaryTest(path, options, print) {
   return printed;
 }
 
+function isMemberChainElement(node) {
```

Comment:
```suggestion
function isChainElement(node) {
```

This is how https://github.com/estree/estree/blob/master/es2020.md#chainexpression call this.